### PR TITLE
Adding kill and replace functionality.

### LINKED ISSUE #61: Feature request: Add a kill flag
I'd like to kill the currently running command and kick off a fresh one if a file changes. This is in constast with -l which just doesn't execute a second command.

### PR DESCRIPTION
**Pull request by placeybordeaux**
This was a rough cut to fix #61. It seems to work largely as expected, although I didn't test all the combinations of flags.

### CODE REVIEW

File: whenchanged/whenchanged.py [hunk 1]
```diff
@@ -30,6 +33,35 @@
     # Standard library
     import subprocess
 
+class CmdRunner():
```
Comment (joh) [inline whenchanged/whenchanged.py]: Having a class for this seems a bit overkill. Can you find a way to integrate this functionality into the WhenChanged class?

**Comment by joh** [comment]
Sorry for taking so long to review this. I like the feature, please have a look at my review comments :)

**Comment by placeybordeaux** [comment]
Hmm I think that this should be considered WIP and not merged till all_pids can never have unrelated pids in it.

**Comment by joh** [comment]
This code doesn't look portable as it is. Make sure this works on Windows + Mac OS X as well. Maybe this can be made portable with subprocess.CREATE_NEW_PROCESS_GROUP?

**Comment by nelson6e65** [comment]
I hope you can continue developing this functionality. :relaxed:

**Comment by placeybordeaux** [comment]
@nelson6e65 I ended up switching to https://github.com/watchexec/watchexec. This PR now has merge conflicts and I won't be taking the time to address the outstanding comments or fixing the merge conflicts. If you want to see this merged please feel free to take this change set & address the concerns!

**Comment by placeybordeaux** [comment]
Closing this pull request as I won't be updating my fork of this repo.

**Pull request closed without merging**